airbyte-integrations/connectors/source-instagram/source_instagram/streams.py

```diff
@@ -143,10 +143,18 @@ def read_records(
 class UserLifetimeInsights(DatetimeTransformerMixin, InstagramStream):
     """Docs: https://developers.facebook.com/docs/instagram-api/reference/ig-user/insights"""
 
-    primary_key = ["business_account_id", "metric", "date"]
```

Comment:
I don't have any sample response from the previous version so I can't confirm what `date` really was. However, if the value was being updated every sync, it would mean that there was a new entry created in the destination every time. This will not be the case anymore. Is this an issue? Should we document this behavior change?

Reply:
You could find previous version [here](https://github.com/airbytehq/airbyte/blob/cc7f01925397c7d773f8cae33eb19651bf697c85/airbyte-integrations/connectors/source-instagram/integration_tests/expected_records.jsonl)
This is an issue, because data will be returned for ALL period, see https://github.com/airbytehq/airbyte/pull/33930/files#diff-ade20df2f7ea130a82ed265a6a96eb70c0d9ccb98e92bb61afdb5b05c146525fR27

Reply:
So basically, we are moving from `metric_type=time_series` to `metric_type= total_value`, right? If so, is this what the customer wants? This seems like a different stream entirely

Reply:
Not really, we'll receive the [most `similar` metrics](https://github.com/airbytehq/airbyte/pull/33930/files#diff-ade20df2f7ea130a82ed265a6a96eb70c0d9ccb98e92bb61afdb5b05c146525fR31) as it was described in [FB docs (alternative metric)](https://developers.facebook.com/docs/instagram-api/reference/ig-user/insights#metrics-and-periods)

Reply:
Yes but they are not over time right?

Reply:
No, they are "lifetime", just 1 record per metric without any date.

Reply:
Now yes, but before, "date" was part of the primary key right? So we could have multiple entries in the database for the same metric (depending on the date)

Reply:
Sync discussion summary: The metric does not support `time_series`. So this is a breaking change and we don't have an easy way around it. We're fine with the change and will try to intercept user feedback if this is not good enough. In the meanwhile, @artem1205 will document the change better in the migration guide.